portal-web/docroot/html/common/themes/password_expiring_soon.jspf

```diff
@@ -14,13 +14,13 @@
  */
 --%>
 
-<c:if test="<%= themeDisplay.isSignedIn() && UserLocalServiceUtil.isPasswordExpiringSoon(user) %>">
-	<aui:script use="liferay-notice">
-		new Liferay.Notice(
+<c:if test='<%= SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

Comment:
I was thinking about it, but my thought process assumed that we would only be in the `LoginPostAction` if we were signed in... is that correct? Also I'm not sure if I can access the themeDisplay in `LoginPostAction`, and I'm not sure how else we can access that information